MeshLib/CoordinateSystem.h

```diff
@@ -47,8 +47,7 @@ class CoordinateSystem
     explicit CoordinateSystem(const Element &ele);
 
     /// Decides a coordinate system from a bounding box
-    template <class T>
-    explicit CoordinateSystem(const GeoLib::AABB<T> &bbox) : _type(getCoordinateSystem(bbox)) {}
+    explicit CoordinateSystem(const GeoLib::AABB &bbox) : _type(getCoordinateSystem(bbox)) {}
```

Comment:
Okay. Maybe I am a bit pedantic here but I still think the name ```CoordinateSystem``` is wrong (i.e. suggesting something the class does not do). We can discuss this later, though, it's not related to this PR.